config/fam_pe_config

```diff
@@ -0,0 +1,19 @@
+# PE config file
```

Comment:
Are you planning to add PE config changes as part of this checkin ? If not leave this file as empty with single line comment saying # PE config file here.